### DISCUSSION TITLE
websocket documentation

### DISCUSSION THREAD

**Discussion by travisghansen** [zwave-js/zwave-js-ui#1875] [Q&A]
Is the websocket api documented anywhere?

**Comment by robertsLando** [comment]
Do you mean the websocket connection I use on control panel UI or zwavejs server websocket? Because the one I use in UI is not documented as I always used it for internal use, the zewavejs server one is much more complete and it's documented AFAIK

**Reply by travisghansen** [reply]
The same as used by hass integration. Where did I miss the docs?

**Reply by robertsLando** [reply]
https://github.com/zwave-js/zwave-js-server

**Reply by robertsLando** [reply]
cc @raman325

**Reply by travisghansen** [reply]
Ah ok! I’ll dig through that and let you know if any further questions arise!
